docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1178,6 +1178,22 @@ How to **investigate**:
   {name="rollout-operator",namespace="<namespace>"}
   ```
 
+### MimirIngestedDataTooFarInTheFuture
+
+This alert fires when Mimir ingester accepts a sample with timestamp that is too far in the future.
+This is typically a result of processing of corrupted message, and it can cause rejection of other samples with timestamp close to "now" (real-world time).
+
+How it **works**:
+
+- The metric exported by ingester computes maximum timestamp from all TSDBs open in ingester.
+- Alert checks this exported metric and fires if maximum timestamp is more than 1h in the future.
+
+How to **investigate**
+
+- Find the tenant with bad sample on ingester's tenants list, where a warning is displayed if Head MaxT is too far in the future.
```

Comment:
This refers to change done in https://github.com/grafana/mimir/pull/5824

Reply:
Switched to "TSDB Head max timestamp too far in the future" message both here, and in tenants listing view.